tests/integration/components/imgix-image-test.js

```diff
@@ -1,85 +1,126 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 
-/* global URI */
-
 moduleForComponent('imgix-image', 'Integration | Component | imgix image', {
   integration: true
 });
 
 test('it renders', function(assert) {
   this.render(hbs`{{imgix-image path="/users/1.png"}}`);
+  assert.ok(this.$());
+});
 
-  assert.equal(this.$().text().trim(), '');
-  assert.equal(this.$('img').length, 1);
+test('it renders event more better', function(assert) {
+  this.render(
+    hbs`<div style='width:200px;height:200px;'>{{imgix-image path='/users/1.png' }}</div>`
+  );
+
+  let url = new URL(this.$('img').attr('src'));
+  assert.equal(
+    this.$()
+      .text()
+      .trim(),
+    ''
+  );
+  assert.equal(url.pathname, '/users/1.png');
 });
 
 test('it renders the correct path', function(assert) {
```

Comment:
This test appears to be checking the parameter rather than the path as the descriptor states, and furthermore it seems to be relying on the "natural" width of the browser to ensure the test executes properly. This seems brittle to me, can we fix this up?